index.md

```diff
@@ -16,7 +16,8 @@
 - [[CTV15]](https://eprint.iacr.org/2015/377)
 - [ZKBoo [GMO16]](https://eprint.iacr.org/2016/163.pdf)
 - [[Groth16]](https://eprint.iacr.org/2016/260.pdf)
-- [Bootle proofs [BCC+16]](https://eprint.iacr.org/2016/263.pdf)
+- [BCC+16](https://eprint.iacr.org/2016/263.pdf)
+- [Bulletproofs [BBBPWM17]](https://web.stanford.edu/~buenz/pubs/bulletproofs.pdf)
```

Comment:
Concept NACK on this change. The intent of this section is to list the papers introducing each proving system design, with performance improvement papers referenced later on in the implementations section.

If we want to rework this section to list *all* papers, then IMHO we should make this a sub-bullet and add the other performance papers as well (like BCTV14a), so we are still making clear the lineage trees.

Reply:
I'm strongly in favor of creating a hierarchy. It will be more helpful, more comprehensive, and respect the fact that it's an ill-defined line between "performance improvement" and a new scheme.

Here's a template (from a different domain) that I'd suggest:
http://www.cs.tau.ac.il/~tromer/cryptodev.html

In any case, "Bootle proofs" should be removed.

Reply:
Yeah, that looks nice. I'll merge this and then do a follow-up pass to refactor it into a hierarchy.